### DISCUSSION TITLE
SNAPSHOT-20231110.205939

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#1018] [General]
* VimR is not using custom build binary using `libnvim` anymore. Thanks to GH-1015 by [georgeharker](https://github.com/georgeharker), VimR now even can use external Neovim binary.
* Neovim 0.10.0-dev
* Minimum macOS requirement is now Ventura (v13).
* Fix VimRSetCharacterspacing: GH-1014
* Add basic file handling to the filebrowser: GH-1005
* Provide neovim original commandline interface: GH-1003
* Add an option for custom Markdown processor: GH-745
* Dependencies updates:
  - ReactiveX/RxSwift@6.6.0
  - sparkle-project/Sparkle@2.5.0
  - apple/swift-collections@1.0.5
  - Quick/Nimble@13.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/snapshot/20231110.205939'>SNAPSHOT-20231110.205939</a>.</em>

**Comment by wookayin** [comment]
Congratulations and great work @qvacua and @georgeharker. 

> VimR now even can use external Neovim binary.

One question -- How is this possible? Also, if using an external neovim, would `has("gui_vimr")` still return 1?

**Reply by georgeharker** [reply]
Currently that isn’t set. But will work on something like it. The has(‘gui_running’) will work.

**Reply by georgeharker** [reply]
I should note g:gui_VimR will be set. 

This works by using messagepack api to Neovim. 

As such because neovim wasn't compiled with a gui in has(gui) reports false. But has(gui_running) will be true as well as g:gui_vimr being 1. 

Tidying up this is a work I. Progress. I'm not sure if new options can be created with an external neovim.
